snap/hooks/install

```diff
@@ -15,33 +15,36 @@ function create_file_structure () {
     dir_copy_if_not_exists "config" "${SNAP_COMMON}/backups" 774
 
     # -------------------------------
+    ETC="${SNAP_DATA}/etc/opensearch/"
+    USR="${SNAP_DATA}/usr/share/opensearch/"
```

Comment:
Could you please remove the trailing slashes, it's usually easier to read without it so we can use it in paths, i.e `${ETC}/...`